doc/tech/usecases.md

```diff
@@ -2,23 +2,46 @@
 # CNTT Use Cases
 
 ## Table of Contents
- * [5.1 Core](#5.1)
- * [5.2 Edge](#5.2)
+ * [5.1 Edge](#5.1)
+   * [5.1.1 Executive Summary](#5.1.1)
+   * [5.1.2 Objective](#5.1.2)
+   * [5.1.3 Approach & Scope](#5.1.3)
+   * [5.1.4 Principle](#5.1.4)
+   * [5.1.5 Use cases](#5.1.5)
+   * [5.1.6 Terminologies](#5.1.6)
+   * [5.1.7 SDO interlocking](#5.1.7)
+
    
 <a name="5.1"></a>
-## 5.1 Core
+## 5.1 Edge
 
+<a name="5.1.1"></a>
+### 5.1.1 Executive Summary
 
-<a name="5.1"></a>
-## 5.1 Edge
+Edge computing consider on of disruptive  technology should be considered by any DSP as it is tightly linked with a lot of use cases that can monetize the 5G adoption , so a lot of CSP , Punblic  cloud , content provider and application developer focus on Edge 
+
+Edge consider one of main enabler to 5G use cases adoption  , as there is a lot of 5G challenges that Edge will support to solve them , So to achieved 5G requirements  that define by ITU-2020 for 5G , To have enhanced mobile broadband (eMBB) , massive machine-type communications (mMTCs) and ultra-reliable low-latency communications (uRLLCs) Edge is required 
```

Comment:
- Edge consider -> Edge is considered
- "as there is a" -> as there are a
- "achieved 5G " -> achieve 5G

Reply:
done

Reply:
@CsatariGergely , can we resolve the conversation